src/app/components/transactions/transactionTitle.tsx

```diff
@@ -30,6 +30,18 @@ export default function TransactionTitle(props: TransactionTitleProps) {
     return tx.incoming ? t('TRANSACTION_RECEIVED') : t('TRANSACTION_SENT');
   };
 
+  const getBtcTokenTransferTitle = (tx: BtcTransactionData): string => {
+    if (tx.isOrdinal && tx.txStatus === 'pending') {
```

Comment:
I think we can make the if-block with `tx.txStatus === 'pending'` check and the inner if-block for `tx.isOrdinal` check, should be easier to read